modules/tinymce/src/plugins/autolink/test/ts/module/test/KeyUtils.ts

```diff
@@ -1,7 +1,10 @@
 import { Arr, Fun, Unicode } from '@ephox/katamari';
-import * as NodeType from 'tinymce/core/dom/NodeType';
 
-const charCodeToKeyCode = (charCode) => {
+import Editor from 'tinymce/core/api/Editor';
+
+const isText = (node: Node): node is Text => node.nodeType === 3;
```

Comment:
Should we be using the sugar version of this?

Reply:
Probably, but nothing here currently uses sugar so converting would be a much larger task and is out of scope.

Reply:
Good point. Should we replace the 3 with `NodeTypes.TEXT` or nah